src/model/immutable/DefaultDraftBlock.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule DefaultDraftBlock
```

Comment:
Could not come up with a better name for it, DefaultDraftBlock ideally DraftBlockMap, but we already use that for other things